WordPress/Classes/ViewRelated/Post/Preview/PreviewWebKitViewController.swift

```diff
@@ -162,7 +172,7 @@ class PreviewWebKitViewController: WebKitViewController {
 
         alertController.addCancelActionWithTitle(cancelTitle)
         alertController.addDefaultActionWithTitle(publishTitle) { [unowned self] _ in
-            PostCoordinator.shared.publish(self.post)
+            PostCoordinator.shared.publish(self.post!)
```

Comment:
This is more a suggestion/personal preference. Thought I don't see a case where `self.post!` can be `nil` I would use a `guard` statement to avoid force unwrapping.

Reply:
Good suggestion, it is definitely better with a guard!